src/components/Asset.vue

```diff
@@ -110,6 +119,7 @@ export default {
         autoplays: this.prefersReducedMotion ? false : this.videoAutoplays,
         posterVariants: this.videoPoster ? this.videoPoster.variants : [],
         deviceFrame: this.deviceFrame,
+        altTextId: this.videoHasAlt ? this.altTextId : null,
```

Comment:
Can we just pass the alt text itself here and render the video specific alt text element in the video component if it exists?

Reply:
Fixed it